fh-mbaas-template-3node.json

```diff
@@ -1677,6 +1677,12 @@
       "value": "info"
     },
     {
+      "name": "MONGODB_PVC_SIZE",
+      "description": "The size of the volume for MongoDB Data",
+      "value": "50Gi",
+      "required": true
+    },
+    {
       "name": "SMTP_SERVER",
```

Comment:
Thanks folks, I'll bump version and merge this before it has to be rebased again. :)